go-carbon.conf.example

```diff
@@ -308,10 +333,13 @@ encoding-time = "iso8601"
 # Log duration format: "seconds", "nanos", "string"
 encoding-duration = "seconds"
 
-[[logging]]
-logger = ""
-file = "stdout"
-level = "error"
-encoding = "mixed"
-encoding-time = "iso8601"
-encoding-duration = "seconds"
+# You can define multiply loggers:
```

Comment:
It was commented out in the README.md, so I just copied the modification to the example config. 
There is already a default logger above, is it useful to have a second logger enabled instead of juste saying that it's available if need be ?